### ISSUE TITLE
Not Working. Instructions hard to follow

### ISSUE THREAD

**Issue by noddynod443** [Iswenzz/Q3-to-CoD4-Map-Converter#1]
Hello, I find the readme instructions very difficult to understand and follow. There is no "BSP" folder and I can not find a converter.exe file to run. Can you please help clarify my misunderstanding?

**Comment by Iswenzz** [comment]
Hello, it seems i didn't release a compiled exe, so you have to run the python file. It will create the BSP folder, or you can just create it.
